Sources/Suggestions/Score.swift

```diff
@@ -104,12 +111,9 @@ extension Score {
     }
 
     static func tokens(from query: Query) -> [Query] {
-        return query
-            .split(whereSeparator: {
-                $0.unicodeScalars.contains(where: { CharacterSet.whitespacesAndNewlines.contains($0) })
```

Comment:
This seemed superfluous and appeared to result in some strings not tokenising as expected, so this is aligned with the similar code for Bookmarks on iOS.